### ISSUE TITLE
Parsing error caused by the new version of MplusAutomation

### ISSUE THREAD

**Issue by andkov** [IALSA/ialsa-2017-portland#3]
It seems that I have been able to track the error I was getting during parsing manually produced output.  The function `readModels` seems to expec the `.dat` file by default and the `extractModelSummaries` are deprecated. 

The outputs that were generated mechanistically are parsing fine (except for LASA, which is a different issue #2)

## Suggested action
- rely on the existing products of parsing generated by an older version of `MplusAutomation`. Unless any discrepancies are found between the catalog and the raw files it aggregates. 
- This is another reason to discontinue support of manually estimated output, @ampiccinin . (Aside: for this round in phys-cog we say we have 9 studies: for 6 of them the outputs were produced by manual estimation using humans running Mplus and for the 3 of them the outputs were produced by estimation, mechanized in R. ([review the list ](https://github.com/IALSA/ialsa-2017-portland/tree/master/model-output/physical-cognitive))  We have the extracted catalogs from all 6 of "manual" studies and we don't plan to re-run any of them anyway. And we can't. So that piece is fixed. The other 3 studies - we can do what we want now (after I fix this version update issue)
- if necessary, the extraction from the manually produced outputs can be re-created by reverting to the older version of  `MplusAutiomation` (I don't know which, that is TBD) 
- moving forward, we focus on studies that provide the chain of custody from the [ialsa-study-curator](https://github.com/IALSA/ialsa-study-curator/blob/master/README.md) to the "estimation factory" of IalsaSynthesis and produce all model output mechanistically. 
- for the immediate time I will focus on putting online what we do have and modularize access to study catalogs.

**Comment by wibeasley** [comment]
@andkov, tell me if you want help with regexes, or any of the functions we wrote.

**Comment by andkov** [comment]
@wibeasley , I might, thank you. However, it's more about `MplusAutomation` version. They have deprecated a lot of functions and changed the way objects are treated. So I don't want to go down that rabbit hole unless absolutely necessary. At this point, i'm happy with the catalogs from the **manual** studies, so if I automate the estimation for LASA, MAP, OCTO relying on  version `0.7` we'll have a working pipeline for three studies and a historical, dead catalog from the original workshop. That, IMHO, should be sufficient to feed this round of publications. Does this sound ok, @ampiccinin ? To clarify, yes, we will re-run these three studies without excluding `dementia_ever = 1` subsample.

**Comment by wibeasley** [comment]
@andkov, from what I understand, I agree with your assessment of the tradeoffs.  If you need a short-term solution, remember you turn back the clock to before the functions were deprecated

### from CRAN, automated
See a combination of these two documents.  Basically, the 'remotes' package was spawned a year ago from parts of the `devtools` package, so they're more focused.
* https://www.rdocumentation.org/packages/remotes/versions/1.1.1/topics/install_version
* https://support.rstudio.com/hc/en-us/articles/219949047-Installing-older-versions-of-packages

### from GitHub  
The [remotes](https://github.com/r-lib/remotes#readme) package will allow you to install an old version of MplusAutomation (and ).

>To install a certain branch or commit or tag, append it to the repo name,  after an @:
>
> `remotes::install_github("gaborcsardi/pkgconfig@v2.0.0")`

### from CRAN, manually,

Download the tarball from CRAN, and install it through RStudio's "install" button.
old tarballs: https://cran.r-project.org/src/contrib/Archive/MplusAutomation/

**Comment by andkov** [comment]
yes, @wibeasley , thanks for the guides, I was thinking undertaking of something like this and the `remotes::` seems great to do just that. I thought I would have to do it via client and it was giving shivers. Thanks for mentioning this. 

 I just want to do it at the right time so I don' t disperse my attention. But at that time, it will come very handy.
